templates/create_project.html

```diff
@@ -37,7 +34,7 @@
     width: 80%;
     padding-left: 20px;
 }
-.nav{
+.nav-body{
```

Comment:
During the weekly meeting you discussed why you are leaving this in the child files, and that is okay. To make sure we do not forget in the future, can you put comments above these .nav-body styles that explain why you chose not to move the .nav-body style to the parent_signed_in file?